main/src/com/google/refine/ProjectMetadata.java

```diff
@@ -368,12 +372,18 @@ private void updateUserMetadata(String metaName, String valueString) {
         }
     }
 
-    public void setAnyField(String metaName, String valueString) {
+    public void setAnyField(String metaName, String valueString) throws JsonMappingException, JsonProcessingException {
         Class<? extends ProjectMetadata> metaClass = this.getClass();
         try {
             Field metaField = metaClass.getDeclaredField("_" + metaName);
             if (metaName.equals("tags")) {
                 metaField.set(this, valueString.split(","));
+            } else if (metaName.equals("customMetadata")) {
+                ObjectMapper customMetadataMapper = new ObjectMapper();
```

Comment:
Instead of defining a new `ObjectMapper` (which is costly), it could be simpler to reuse `ParsingUtilities.mapper`.